core/src/http_server/routing.rs

```diff
@@ -0,0 +1,56 @@
+//! Module implementing the default HTTP router for the monitoring endpoints.
+
+use super::Handler;
+use crate::metrics::MetricsHandler;
+use anyhow::Result;
+use rouille::{router, Request, Response};
+
+pub struct DefaultRouter(pub MetricsHandler);
```

Comment:
Maybe this should become a builder in the future so that we can incrementally pass more handlers into it as we build them in our main.rs?

Reply:
I already changed this in the next PR so you can create it with handlers by name:
https://github.com/gnosis/dex-services/blob/d317ba253d29ac24f948708a13bf5176c0340582/driver/src/main.rs#L350-L354

I'll merge as is for now, and we can discuss how we want to build the router in the next PR.